includes/google_scholar.inc

```diff
@@ -44,36 +43,46 @@ function islandora_scholar_create_meta_tags($object) {
     else {
       return FALSE;
     }
-    foreach ($mods_xml->xpath('mods:name') as $name_xml) {
+    foreach ($mods_xml->xpath(variable_get('islandora_scholar_xpaths_authors_xpath', 'mods:name')) as $name_xml) {
```

Comment:
This is good, we should definitely not be using the hardcoded xpath if a configurable one exists.